doc/index.rst

```diff
@@ -1,9 +1,32 @@
-The Plutus Platform
+Plutus Tools Developer Guide
 ===============================
 
-The Plutus Platform is an application development platform for the Cardano blockchain.
+Plutus Tools are a collection of off-chain infrastructure resources. Plutus Tools are built 
+for developers who are authoring web-based applications that implement smart contracts on the 
+Cardano blockchain. 
+
+The Plutus Tools Team supports the underlying resources that developers need who are 
+writing full applications using Plutus in Haskell, including off-chain code. 
+The term “off-chain code” refers to the part of a contract application’s code which 
```

Comment:
Should this just link to the glossary?

Reply:
Good point. However, I want the reader to have a sufficient general understanding of this concept without needing to leave the landing page.